package.json

```diff
@@ -30,6 +30,7 @@
     "@storybook/react": "8.2.9",
     "@storybook/theming": "8.2.9",
     "@svgr/webpack": "8.1.0",
+    "@tailwindcss/container-queries": "^0.1.1",
```

Comment:
Is this dep needed for the storybook only?

Reply:
ah, I see it's required as a peer dependency for @guild/components, maybe better to add it as a dependency in `@theguild/tailwind-config`?

Reply:
Hmm, makes sense. In general, is this something you'd consider useful globally?

Reply:
can you remove this from dependencies and use `0.6.0-alpha-20241106003557-2c94714753ba2342f442f836b4683119985ee2c5` version of `@the-guild/tailwind-config`? (I am trying to do official release asap of the tailwind config). Have some problems now that it include all versions and doesn't update changelogs.. https://github.com/the-guild-org/shared-config/pull/620

Reply:
bumping to 0.6.0-alpha-20241106003557-2c94714753ba2342f442f836b4683119985ee2c5